### ISSUE TITLE
Pandora does not work properly.

### ISSUE THREAD

**Issue by buffet-time** [pendo324/universal-np-extension#16]
So for me Pandora isn't working this is likely due to the fact that Pandora Premium functions a bit different than non Premium Pandora. So the issue is that you're only looking at what works for the Free or Plus Version. Need to add some handling for Premium which it's either at the bottom of the page or it doesnt show (but the div is still there) 

```
<a class="Tuner__Audio__TrackDetail__title" data-qa="mini_track_title" href="/artist/cavetown/16-04-16/bg-noise/TRnw23fbpV5kKxq">Bg Noise</a>
 
<a class="Tuner__Audio__TrackDetail__artist" data-qa="mini_track_artist_name" href="/artist/cavetown/ARg6tcm5Ph9fz2J">Cavetown</a>
```

These can be used to get the Track Title and Artist Name on every page from what i can see.

**Comment by buffet-time** [comment]
Submitted a PR

https://github.com/pendo324/universal-np-extension/pull/17

**Comment by pendo324** [comment]
Merged #17. Bumped the version and published to chrome webstore. It's in review, don't know how long that will take (hopefully not too long). Thanks for the PR!
